src/Engine/Core/Fusee.Engine.Core.csproj

```diff
@@ -44,5 +44,6 @@
     <ProjectReference Include="$(FuseeEngineRoot)\src\Serialization\Fusee.Serialization.csproj">
       <PrivateAssets>analyzers</PrivateAssets>
     </ProjectReference>
+    <ProjectReference Include="$(FuseeEngineRoot)\src\Structures\Fusee.Structures.csproj" />
```

Comment:
Yes, this is missing. This is not for analyzers to be private, but so contentfiles and build are not private. https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets

This might not be entirely necessary for this project but we are using this as a default throughout the repository.